testassets/LinkerTestsClient/LinkerTestsClient.csproj

```diff
@@ -1,29 +1,18 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <OutputType>Exe</OutputType>
-    
     <PublishTrimmed>true</PublishTrimmed>
-    <TrimMode>Link</TrimMode>
   </PropertyGroup>
 
   <ItemGroup>
     <Protobuf Include="..\Proto\greet.proto" GrpcServices="Client" Link="Protos\greet.proto" />
     <Protobuf Include="..\Proto\unimplemented.proto" GrpcServices="Client" Link="Protos\unimplemented.proto" />
 
-    <PackageReference Include="Google.Protobuf" Version="$(GoogleProtobufPackageVersion)" />
-    <PackageReference Include="Grpc.Net.Client" Version="$(GrpcDotNetPackageVersion)" />
+    <ProjectReference Include="C:\Users\andy\Code\protobuf\csharp\src\Google.Protobuf\Google.Protobuf.csproj" />
+    <ProjectReference Include="..\..\src\Grpc.Net.Client\Grpc.Net.Client.csproj" />
     <PackageReference Include="Grpc.Tools" Version="$(GrpcPackageVersion)" PrivateAssets="All" />
   </ItemGroup>
 
-  <Target Name="EnsureAllAssembliesAreLinked" BeforeTargets="PrepareForILLink">
-    <ItemGroup>
-      <ManagedAssemblyToLink>
-        <TrimMode>link</TrimMode>
-      </ManagedAssemblyToLink>
-      <TrimmerRootAssembly Include="$(TargetName)" />
-    </ItemGroup>
-  </Target>
```

Comment:
No longer needed in .NET 6?

Reply:
`IsTrimmable` does the same thing on a per-assembly basis. I would opt-in in the assembly itself, to enable trimming for everyone